### ISSUE TITLE
PatchTST pretrain

### ISSUE THREAD

**Issue by chenkovsky** [timeseriesAI/tsai#873]
does this library support PatchTST pretrain?

**Comment by SSHurtado** [comment]
> *This was drafted with AI support based on human feedback collected during triage.*

No, tsai does not currently provide a supported PatchTST pretraining workflow.

The current `PatchTST` implementation in tsai is documented as a long-term forecasting model with output shaped like `[batch, variables, pred_dim]`: https://timeseriesai.github.io/tsai/models.patchtst.html. Checking the library implementation also shows no `pretrain`, `pretrained`, or `custom_head` path for PatchTST; the class is built around the supervised forecasting head.

For now, the supported path is supervised training/fine-tuning with `PatchTST`. If you need PatchTST-style pretraining, it would need to be implemented as custom user code, for example by writing a custom pretraining loop or callback and then fine-tuning the resulting model weights for the downstream forecasting task.

Closing this as answered for now.
